packages/pwa-kit-extension-sdk/src/configs/babel/plugin-application-extensions.ts

```diff
@@ -10,14 +10,16 @@
 const parser = require('@babel/parser')
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const babel = require('@babel/core')
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const path = require('path')
 
 // Local
 import {renderTemplate} from '../utils'
 import {buildAliases} from '../../shared/utils'
 import {ApplicationExtensionsLoaderOptions} from '../webpack/types'
 
 // Constants
-const extensionsPlaceholderFile = 'express/placeholders/application-extensions.js'
+const extensionsPlaceholderFile = path.join('express', 'placeholders', 'application-extensions.js')
```

Comment:
This was a real problem that was stopping things from working on windows. Simple fix is to use the os specific separator.